core/conversion/converters/converter_util.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <map>
+#include <string>
+
+#include "core/conversion/conversionctx/ConversionCtx.h"
+#include "core/conversion/converters/Weights.h"
+#include "core/conversion/var/Var.h"
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace conversion {
+namespace converters {
+namespace util {
+    using namespace core::util;
```

Comment:
If I remove this, then all existing util:: references in converters code would have to be changed to core::util::.

Reply:
We usually use `util::` for all utility operations (like unPadDims, toVec etc) when we use them in converters. We also try to avoid using `using namespaces` generally.